src/app/macos/Flutter/GeneratedPluginRegistrant.swift

```diff
@@ -5,8 +5,10 @@
 import FlutterMacOS
 import Foundation
 
+import package_info_plus
 import shared_preferences_foundation
 
 func RegisterGeneratedPlugins(registry: FlutterPluginRegistry) {
+  FPPPackageInfoPlusPlugin.register(with: registry.registrar(forPlugin: "FPPPackageInfoPlusPlugin"))
```

Comment:
shouldn't those file be part of `.gitignore`?

Reply:
Yeah, you're right